releaser composite action: doesn't work on windows runner

On 'windows-2022'

```
Run D:\a\_actions\pyTooling\Actions\main\releaser/composite/../releaser.py
  D:\a\_actions\pyTooling\Actions\main\releaser/composite/../releaser.py
  shell: C:\Program Files\Git\bin\bash.EXE --noprofile --norc -e -o pipefail {0}
  env:
    pythonLocation: C:\hostedtoolcache\windows\Python\3.8.10\x64
    MSYSTEM: MSYS
    INPUT_TOKEN: ***
    INPUT_FILES: staging/mingw/clangarm64/*
    INPUT_TAG: repo
    INPUT_RM: true
    INPUT_SNAPSHOTS: true
D:\a\_temp\193f3dee-d513-4616-810b-1c5c0c0b4d76.sh: line 1: D:a_actionspyToolingActionsmainreleaser/composite/../releaser.py: No such file or directory
Error: Process completed with exit code 127.
```

--- a/releaser/composite/action.yml
+++ b/releaser/composite/action.yml
@@ -48,7 +48,7 @@ runs:
     run: pip install PyGithub --progress-bar off
 
   - shell: bash
-    run: ${{ github.action_path }}/../releaser.py
+    run: '''${{ github.action_path }}/../releaser.py'''
     env:
       INPUT_TOKEN: ${{ inputs.token }}
       INPUT_FILES: ${{ inputs.files }}

